# Representation of a (desired) pose and velocity 
std_msgs/Header header
std_msgs/Float64 time
geometry_msgs/Pose pose
geometry_msgs/Twist twist
geometry_msgs/Twist twd